e2e/scripts/smoke_test.sh

```diff
@@ -41,7 +41,7 @@ rook_infra::create() {
             return 1
         else
             sleep 1
-            attempt_num++
+            ((attempt_num++))
```

Comment:
This might raise an error in `set -e` mode. Its one of those bashisms. I think you should use `(( ++attempt_num ))` instead. See https://unix.stackexchange.com/questions/146773/why-bash-increment-n-0n-return-error